py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -190,6 +216,11 @@ def __init__(
         serial_number = device_info["serial_number"]
         self._transport = transport
 
+        if device_info["product_string"] == BITBOX02MULTI:
+            self.edition = BitBox02Edition.MULTI
+        elif device_info["product_string"] == BITBOX02BTC:
+            self.edition = BitBox02Edition.BTCONLY
+
```

Comment:
else raise exception

Reply:
then I'm raising an exception whenever there is a base?

Reply:
ahhh good point, nvm then